kornia/augmentation/augmentation.py

```diff
@@ -1292,14 +1421,13 @@ def __init__(
     def __repr__(self) -> str:
         return self.__class__.__name__ + f"({super().__repr__()})"
 
-    def generate_parameters(self, batch_shape: torch.Size) -> Dict[str, torch.Tensor]:
-        return dict()
-
     def compute_transformation(self, input: torch.Tensor, params: Dict[str, torch.Tensor]) -> torch.Tensor:
-        return F.compute_intensity_transformation(input)
+        return self.identity_matrix(input)
 
-    def apply_transform(self, input: torch.Tensor, params: Dict[str, torch.Tensor]) -> torch.Tensor:
-        return F.apply_equalize(input, params)
+    def apply_transform(
+        self, input: torch.Tensor, params: Dict[str, torch.Tensor], transform: Optional[torch.Tensor] = None
+    ) -> torch.Tensor:
+        return equalize(input)
 
 
 class GaussianBlur(AugmentationBase2D):
```

Comment:
can we rename that one to `RandomGaussianBlur` ?

Reply:
Probably in the next version. I will add a TODO here.